src/modules/RA/Viewpoint/components/ViewpointFields.js

```diff
@@ -149,6 +149,7 @@ const ViewpointFields = ({
       {...props}
     >
       <FormTab label="resources.viewpoint.tabs.general">
+        <BooleanInput source="active" />
```

Comment:
Is it easy to add something to show that the viewpoint is not visible in the list display ?

Reply:
Like a stroked eye icon on top of the image

Reply:
Yes, good idea.

Reply:
Done.